src/test/java/com/teragrep/cfe_35/router/Output.java

```diff
@@ -178,6 +179,7 @@ public void accept(byte[] syslogMessage) {
                     allSent = true;
                 }
             }
+            LOGGER.info("sent successfully syslogMessage <[{}]>", new String(syslogMessage, StandardCharsets.UTF_8));
```

Comment:
Is it printing everything it sends on info level? Turn that to trace and guard if so